### DISCUSSION TITLE
macOS swap lines doesn't work

### DISCUSSION THREAD

**Discussion by rbruno95** [cpeditor/cpeditor#862] [Q&A] [answered]
Hi, great app! Yesterday I installed CP editor and noticed the following issue

When I try to swap lines with the command that CP Editor has configured in macOS (Command+Shift+(Up or Down Arrow)) it doesn't work, because in macOS, the shortcut Command+Up Arrow for example when used in text, moves the cursor from the current cursor position to the beginning, and Command+Down to the end

So when you also use Shift, what you get is that you select all the way to the beginning or all the way to the end, depending on the direction you have chosen.

The thing is that this shortcut is native to macOS and takes precedence over CP Editor, i.e. it never swaps lines.

Am I the only one who has this problem? If not, what is the possibility to change the shortcut by another key combination like in Sublime Text(Command+Cntrl+Up arrow) or VSCode(Alt+Up arrow)

**Comment by ouuan** [comment, marked as answer]
Thanks for your reporting! I opened #863 for this.

**Reply by rbruno95** [reply]
Cool! Thanks
